src/Microsoft.ML.Data/Transforms/NormalizerCatalog.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.Data.Transforms
+{
+    /// <summary>
+    /// Extensions for normalizer operations.
+    /// </summary>
+    public static class NormalizerCatalog
```

Comment:
What are the use cases for `NormalizerCatalog`? I don't see any usages of it in this PR.

I see it is a static class, while all the other "catalog" classes are instance members of MLContext. Should it be designed the same as say `TransformsCatalog`?

Also, I don't see any tests using this class.

Reply:
Ah, I get it - this provides extension methods to the `TransformsCatalog`. How about we name these "Extension" only classes differently then? It is confusing in my opinion that `NormalizerCatalog` and `TransformsCatalog` are named similarly, but are designed differently.

---
In reply to: [226110284](https://github.com/dotnet/machinelearning/pull/1252#discussion_r226110284) [](ancestors = 226110284)

Reply:
This is a set of extensions to `TransformsCatalog`. I guess I will need to rename it

---
In reply to: [226110284](https://github.com/dotnet/machinelearning/pull/1252#discussion_r226110284) [](ancestors = 226110284)

Reply:
And add tests :D

---
In reply to: [226110673](https://github.com/dotnet/machinelearning/pull/1252#discussion_r226110673) [](ancestors = 226110673,226110284)